server/jetstream.go

```diff
@@ -86,6 +86,7 @@ type JetStreamTier struct {
 type JetStreamAccountStats struct {
 	JetStreamTier                          // in case tiers are used, reflects totals with limits not set
 	Domain        string                   `json:"domain,omitempty"`
+	APILevel      string                   `json:"api_level"`
```

Comment:
Should this be under API instead as just level?

Reply:
Didn't do it at first since `API` contains dynamic stats like total and errored requests, and the `JSApiLevel` is static/hardcoded just like the server `VERSION` is.

The fields have the same prefix and it actually simplifies the code as well when moving it in, though.
Assuming it's still okay, I've moved it into `API.Level`.